### PR TITLE
fix: pages default local port regression

### LINKED ISSUE #5534: 🐛 BUG: regression/breaking change in default port with `pages dev`
### Which Cloudflare product(s) does this pertain to?

Wrangler core

### What version(s) of the tool(s) are you using?

3.47.1

### What version of Node are you using?

20.12.1

### What operating system and version are you using?

Windows 11

### Observed behavior

In wrangler <3.38.0, `wrangler pages dev public` would spawn a server on port `8788` by default.
`[wrangler:inf] Ready on http://127.0.0.1:8788`

In wrangler 3.38.0 and above, the same command spawns the server on port `8787` by default:
`[wrangler:inf] Ready on http://127.0.0.1:8787`

This has broken some workflows for people that rely on it in testing or other workflows.

### Expected behavior
The default port should remain 8788 to avoid issues with documentation, testing, and other things that rely on this, otherwise it's a breaking change.

### Notes

This was introduced in this commit: https://github.com/cloudflare/workers-sdk/commit/f5e2367288e7f57365ef8a1373bbc404bb50a662#diff-c90ff1ca6d929baaafb7d67d6d6ccd573593762adf4e7a9b6917343ceeceb68bL116

The default was seemingly removed: 
image

### PR DESCRIPTION
**Pull request by Cherry**
## What this PR solves / how to test

Fixes #5534

## Author has addressed the following

- Tests
  - [ ] TODO (before merge)
  - [ ] Included
  - [x] Not necessary because: no test coverage for this
- Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))
  - [ ] TODO (before merge)
  - [x] Included
  - [ ] Not necessary because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): <https://github.com/cloudflare/cloudflare-docs/pull/>...
  - [x] Not necessary because: this brings things back inline with what is already reported by documentation

### CODE REVIEW

File: packages/wrangler/src/pages/dev.ts [hunk 1]
```diff
@@ -42,6 +41,7 @@ import type {
 } from "../yargs-types";
 import type { RoutesJSONSpec } from "./functions/routes-transformation";
 
+export const DEFAULT_PAGES_LOCAL_PORT = 8788;
```
Comment (CarmenPopoviciu) [inline packages/wrangler/src/pages/dev.ts:44]: can we please add this on line `78` instead 🙏, just before 👇  ? That way we'll have all these related vars grouped together

```
const DEFAULT_IP = process.platform === "win32" ? "127.0.0.1" : "localhost";
const DEFAULT_SCRIPT_PATH = "_worker.js";
```

Comment (Cherry) [reply packages/wrangler/src/pages/dev.ts]: Sure thing, thanks @CarmenPopoviciu. Done!

Comment (CarmenPopoviciu) [reply packages/wrangler/src/pages/dev.ts]: thank you so much!

**Review by CarmenPopoviciu — APPROVED** [review]

**Comment by CarmenPopoviciu** [comment]
thank you @Cherry for spotting this regression and fixing it <3. These changes will go out with the release tomorrow. Merging!

**Pull request merged**